src/aiida_wannier90_workflows/utils/projectors/fit_hydrogenics.py

```diff
@@ -0,0 +1,211 @@
+from upf_tools.projectors import Projector
```

Comment:
Since we haven't added `upf_tools` to the dependencies and I guess we don't actually import these part of code anywhere inside `src/`, we should move this folder `tils/projectors` to somewhere else, e.g. `aiida-wannier90-workflows/dev`?